### ISSUE TITLE
getButtonAccept() returns null

### ISSUE THREAD

**Issue by Remie-l** [navasmdc/MaterialDesignLibrary#175]
getButtonAccept() returns null if it called before dialog.show() function.

**Comment by sohayb** [comment]
I guess this is a normal behavior since the view is not inflated yet.

**Comment by Remie-l** [comment]
Ерут how can i change text on accept button?

**Comment by sohayb** [comment]
This is how I'm doing it:
````java
            Dialog dialog = new Dialog(getActivity(), getString(R.string.confirmation),
                    getString(R.string.playlist_delete_confirmation));
            dialog.setOnAcceptButtonClickListener(new OnClickListener() {
                @Override
                public void onClick(View v) {
//Some action
                }
            });
            dialog.show();
            ButtonFlat acceptButton = dialog.getButtonAccept();
            acceptButton.setText(getString(android.R.string.ok));
````
hope it'll fix your problem :)

**Comment by Remie-l** [comment]
Yeah, wright. But I think we both know, that changing button after it was shown is not a good idea.

**Comment by sohayb** [comment]
I don't know from where did you get that idea from, but in any case it's getting the job done for now anyway.

**Comment by sphairo** [comment]
This code return null on cancelButton: 

Dialog dialog = new Dialog(MainActivity.this, "Hola mundo", "Este es mi primer cuadro de dialogo!");
                dialog.setOnAcceptButtonClickListener(new View.OnClickListener() {
                    @Override
                    public void onClick(View v) {
                    }
                });
                dialog.setOnCancelButtonClickListener(new View.OnClickListener() {
                    @Override
                    public void onClick(View v) {
                    }
                });
                dialog.show();
                ButtonFlat acceptButton = dialog.getButtonAccept();
                acceptButton.setText("Aceptar");

                ButtonFlat cancelButton = dialog.getButtonCancel();
                cancelButton.setText("Cancelar");

I need help!
